### ISSUE TITLE
.editorconfig is not a valid INI file with root specified

### ISSUE THREAD

**Issue by danil51608** [macisamuele/language-formatters-pre-commit-hooks#184]
The hook works well if *root* is not present in my .editconfig file. However it breaks with the below message if I add *root = true* to my .editconfig:

```shell
Pretty format INI....................................................................Failed
- hook id: pretty-format-ini
- exit code: 1

Input File .editorconfig is not a valid INI file: File contains no section headers.
file: '<string>', line: 1
'root = true\n'
```

My .editconfig
```ini
root = true

[*]
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
charset = utf-8
indent_style = space
indent_size = 4
max_line_length = 80
```

**Comment by danil51608** [comment]
@Delgan Hi

I found out you had updated the [config-formatter](https://github.com/Delgan/config-formatter). Sadly it doesn't fix the issue. The error is still present.

**Comment by Delgan** [comment]
Actually, I haven't updated `config-formatter` yet, so it's not surprising that the issue isn't fixed.

My library depends on another one, namely `ConfigUpdater`, and I was waiting for an answer in https://github.com/pyscaffold/configupdater/issues/123 to decide how I should implement the fix for `.editorconfig` files.

**Comment by danil51608** [comment]
oh, my bad. I thought you updated it with [this](https://github.com/Delgan/config-formatter/commit/ac8db6309b88864ef7fcb8ef5fa1228ac1b69fc9) commit :)

**Comment by Delgan** [comment]
Hey @danil51608.

I just published version `1.2.0` of `config-formatter` which now supports INI files with no section headers.

If you force re-installation of `language-formatters-pre-commit-hooks` by clearing the cache of `pre-commit`, your `.editorconfig` should now be formatted as expected without error. :+1:

**Comment by danil51608** [comment]
@Delgan Hey! Thanks a lot! Works great! 😊
